src/adapter/src/catalog.rs

```diff
@@ -1411,6 +1411,8 @@ impl<S: Append> Catalog<S> {
                     session_id: Uuid::new_v4(),
                     build_info: config.build_info,
                     timestamp_frequency: Duration::from_secs(1),
+                    // TODO: Make this more configurable?
+                    storage_metrics_frequency: Duration::from_secs(120),
```

Comment:
I assume you're just following `timestamp_frequency`, but that's been one of our more embarrassing parameters, because it's actually an interval, not a frequency! It's slated to get renamed to `timestamp_granularity` soon. So `storage_metrics_granularity` or `storage_metrics_collection_interval` instead would be great.